examples/api-routes-graphql/pages/index.js

```diff
@@ -0,0 +1,27 @@
+import fetch from 'isomorphic-unfetch'
+
+const Index = ({ users }) => (
+  <div>
+    {users.map((user, i) => (
+      <div key={i}>{user.name}</div>
+    ))}
+  </div>
+)
+
+Index.getInitialProps = async () => {
+  const response = await fetch('http://localhost:3000/api/graphql', {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json'
+    },
+    body: JSON.stringify({ query: '{users{name}}' })
```

Comment:
```suggestion
    body: JSON.stringify({ query: '{ users { name } }' })
```
is less confusing this way.